### PR TITLE
Starting Fresh. Commit 1 with updated changes.

### PR DESCRIPTION
**Pull request by dtembe**

### CODE REVIEW

**Comment by dtembe** [comment]
Re-forked from RMOTR files and re-merged

**Comment by dtembe** [comment]
Updated Book ID and Author ID Increment counters

**Comment by santiagobasulto** [comment]
This PR will be reviewed by @iamnotmichael

File: bookstore/bookstore.py [hunk 1]
```diff
@@ -1,29 +1,81 @@
...
+#    bookstore['last_author_id'] += 1
+#    author = {
+#        'name': name,
+#        'nationality': nationality,
+#        'id': bookstore['last_author_id']
+#    }
+#    bookstore['authors'].append(author)
+#    return author
+#
+
```
Comment (iamnotmichael) [inline bookstore/bookstore.py]: Good job. There is a lot of comments from hints that make it slightly confusing where it almost looks like you have two def create_bookstore functions. I like the addition of the id fields.

File: bookstore/bookstore.py [hunk 2]
```diff
@@ -1,29 +1,81 @@
...
 
 def add_author(bookstore, name, nationality):
-    pass
+    bookstore['author_id'] += 1
+    author = {'name': name, 'nationality': nationality, 'id': bookstore['author_id']}
+    bookstore['authors'].append(author)
+    return author
```
Comment (iamnotmichael) [inline bookstore/bookstore.py]: Straight forward and to the point. I like the increment part which is something I wish I had time to add.

File: bookstore/bookstore.py [hunk 3]
```diff
@@ -1,29 +1,81 @@
...
+            return b
 
 
 def get_books_by_author(bookstore, author_id):
-    pass
+    books_by_author_list = []
+    for book in bookstore['books']:
+        if author_id == book['author_id']:
+            books_by_author_list.append(book)
+    return books_by_author_list
```
Comment (iamnotmichael) [inline bookstore/bookstore.py]: Everything looks good. Our code is basically the same minus my name/strip/lower id vs your id increment #s.

**Pull request closed without merging**